airflow/contrib/hooks/datastore_hook.py

```diff
@@ -19,58 +19,73 @@
 #
 
 import time
+
 from googleapiclient.discovery import build
+
 from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
 
 
 class DatastoreHook(GoogleCloudBaseHook):
     """
-    Interact with Google Cloud Datastore. This hook uses the Google Cloud Platform
-    connection.
+    Interact with Google Cloud Datastore. This hook uses the Google Cloud Platform connection.
 
     This object is not threads safe. If you want to make multiple requests
     simultaneously, you will need to create a hook per thread.
+
+    :param version: The version of the API it is going to connect to.
+    :type version: str
     """
 
-    def __init__(self,
-                 datastore_conn_id='google_cloud_datastore_default',
-                 delegate_to=None):
+    def __init__(self, datastore_conn_id='google_cloud_datastore_default', delegate_to=None, version='v1'):
```

Comment:
Other hooks use order `api_version`, `gcp_conn_id`, `delegate_to`.
They also use the name of the `api_version` instead name of `version`.

I plan to standardize parameter names in operators and hooks in the future. The task is waiting in the queue

Reply:
Agree. I like `api_version` more. I've changed it.